dev/run-tests-jenkins.py

```diff
@@ -232,6 +234,10 @@ def main():
     result_message = github_message('has finished')
     result_message += '\n' + test_result_note + '\n'
     result_message += '\n'.join(pr_check_results)
+    result_message += '\n' + ' * This patch tested by `%s` with `%s` and `%s`.' % (
+        os.environ.get("AMPLAB_JENKINS_BUILD_TOOL", "sbt"),
+        os.environ.get("AMPLAB_JENKINS_BUILD_PROFILE", "hadoop2.7"),
```

Comment:
one nit is though we should change this when we change default profile. Might better note it at `pom.xml` so that they can be updated together.